### PR TITLE
support warmup from inside of container

### PR DESCRIPTION
**Pull request by davies**

### CODE REVIEW

File: cmd/rmr.go [hunk 1]
```diff
@@ -44,16 +43,22 @@ $ juicefs rmr /mnt/jfs/foo`,
...
+	st, err := os.Stat(mp)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	if !st.IsDir() {
+		mp = filepath.Dir(mp)
 	}
-	if err != nil && path != "/" {
-		return openController(filepath.Dir(path))
+	for ; mp != "/"; mp = filepath.Dir(mp) {
```
Comment (SandyXSD) [inline cmd/rmr.go:54]: With the change in vfs we can access .control under any directory in juicefs, so we don't need the for loop any more.

File: cmd/warmup.go [hunk 2]
```diff
@@ -150,59 +151,52 @@ func warmup(ctx *cli.Context) error {
...
-			index++
+		if mp == "/" {
+			inode, err := utils.GetFileInode(path)
+			if err != nil {
+				logger.Errorf("lookup inode for %s: %s", mp, err)
+				continue
+			}
+			batch = append(batch, fmt.Sprintf("inode:%d", inode))
+		} else if strings.HasPrefix(path, mp) {
+			batch = append(batch, string(path[start:]))
```
Comment (SandyXSD) [inline cmd/warmup.go:186]: path is already a string

File: cmd/rmr.go (hunk 1 above)
Comment (davies) [reply cmd/rmr.go:54]: This loop is used to find out the mount point to pass relative pass into client.

Comment (SandyXSD) [reply cmd/rmr.go:54]: mp is not returned in this func

Comment (davies) [reply cmd/rmr.go:54]: For backward compatibility, new version should work with existing mount point.

**Review by SandyXSD — APPROVED** [review]

**Pull request merged**